core/tests/protractor_desktop/embedding.js

```diff
@@ -258,7 +258,10 @@ describe('Embedding', function() {
         by.css('.protractor-test-embedded-exploration > iframe')));
       waitFor.pageToFullyLoad();
 
-      // TODO(kevinlee12): Remove this sleep.
+      // TODO(#7622): Remove this sleep. Remove the todo in
```

Comment:
The todo in the second sentence had to be lower case or it would have triggered the lint error.

Reply:
OK -- might need to do this in scripts/pre_commit_linter.py as well? Not sure.

Reply:
no need, it passes the linter :)